pkg/kubelet/config/defaults.go

```diff
@@ -26,4 +26,5 @@ const (
 	DefaultKubeletContainersDirName          = "containers"
 	DefaultKubeletPluginContainersDirName    = "plugin-containers"
 	DefaultKubeletPodResourcesDirName        = "pod-resources"
+	DefaultKubeletPluginsDirSELinuxLabel     = "system_u:object_r:container_file_t:s0"
```

Comment:
Is this label documented anywhere? Should it be?

Reply:
@tallclair no, i think nowhere in k8s documentation.
Should it be? ---- Documentation never hurts. I am just not sure where would it be appropriate to mention about this. Any suggestions please?

Reply:
Sounds good to me.

Reply:
Will update proposals post merge of this PR